modules/ROOT/pages/blogs/entry/announce-apache-netbeans-20-released.adoc

```diff
@@ -0,0 +1,55 @@
+// 
+//     Licensed to the Apache Software Foundation (ASF) under one
+//     or more contributor license agreements.  See the NOTICE file
+//     distributed with this work for additional information
+//     regarding copyright ownership.  The ASF licenses this file
+//     to you under the Apache License, Version 2.0 (the
+//     "License"); you may not use this file except in compliance
+//     with the License.  You may obtain a copy of the License at
+// 
+//       http://www.apache.org/licenses/LICENSE-2.0
+// 
+//     Unless required by applicable law or agreed to in writing,
+//     software distributed under the License is distributed on an
+//     "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+//     KIND, either express or implied.  See the License for the
+//     specific language governing permissions and limitations
+//     under the License.
+//
+
+= [ANNOUNCE] Apache NetBeans 20 Released
+:author: Geertjan Wielenga
+:page-revdate: 2023-12-01
+:page-layout: blogentry
+:page-tags: blogentry
+:jbake-status: published
+:keywords: Apache NetBeans 20 release
+:description: Apache NetBeans 20 release
+:toc: left
+:toc-title:
+:page-syntax: true
+
+
+The Apache NetBeans team is pleased to announce that Apache NetBeans 20 was released on December 1, 2023.
+
+What's in the Apache NetBeans 20 release:
+
+https://github.com/apache/netbeans/releases/tag/20
+
+With thanks to 26 contributors, including 6 who have contributed for the first time. Thank you!
```

Comment:
esp thanks to dependabot who made his first contribution

Reply:
was not already there ?

Reply:
no its the first time that we merged a dependabot PR :P